lib/actions/appleMusic.js

```diff
@@ -58,6 +191,28 @@ function appleMusic(player, values) {
   }
 }
 
+function httpreq(options, responseCallback) {
```

Comment:
You could consider using the already dependant library "request-promise" which would take care of http/https and JSON-parsing directly.